<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Life" failures="0" tests="13" name="priorityoftestcases.PriorityOfTestCases" time="4.023" errors="1" timestamp="2022-09-29T21:51:10 IST" skipped="1">
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test8" time="2.004">
    <error message="Method priorityoftestcases.PriorityOfTestCases.Test8() didn&#039;t finish within the time-out 2000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method priorityoftestcases.PriorityOfTestCases.Test8() didn't finish within the time-out 2000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:384)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:283)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Test8 -->
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test2" time="0.001"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test6" time="0.001"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test6" time="0.001"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test6" time="0.002"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test9" time="2.004"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test4" time="0.001"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test7" time="0.004"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test6" time="0.001"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test6" time="0.001"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test1" time="0.002"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test3" time="0.001"/>
  <system-out/>
  <testcase classname="priorityoftestcases.PriorityOfTestCases" name="Test5">
    <skipped/>
  </testcase> <!-- Test5 -->
  <system-out/>
</testsuite> <!-- priorityoftestcases.PriorityOfTestCases -->
